server/server.js

```diff
@@ -100,20 +97,15 @@ systemRoute.get('system.monitor', _addProxy('/_monitor'), System.monitor)
 systemRoute.get('system.about', _addProxy('/_about'), System.about)
 systemRoute.get('system.paths', _addProxy('/_paths'), System.paths)
 systemRoute.get('system.swaggerUI', _addProxy('/swagger/swagger-initializer.js'), System.swaggerUI)
-systemRoute.get('system.swaggerUI', _addProxy('/swagger'), System.swaggerUI)
```

Comment:
I feel I have to learn more about our swagger-setup. Why do we have so many swagger-routes? 
I usually access swagger through e.g. `api/kursinfo/swagger/index.html`, will this still work when removing `/swagger`?

Reply:
Yes it will still work as we are adding `/swagger` again a few lines below this :) 

I'm not sure why but `/api/kurs-pm/swagger/` didn't work before this change and after both `/api/kurs-pm/swagger/index.html` and `/api/kurs-pm/swagger/` works fine.